source/_components/mailgun.markdown

```diff
@@ -27,8 +27,18 @@ mailgun:
   api_key: XXXXXXXXXXXXX
 ```
 
-Configuration variables:
-
-- **domain** (*Required*): This is the domain name to be used when sending out mail. Defaults to the first custom domain you have set up.
-- **api_key** (*Required*): This is the API token that has been generated in your Mailgun account.
-- **sandbox** (*Deprecated*): Whether to use the sandboxed domain for outgoing mail. Since the `domain` item is required, it should be set to the sandbox domain name, so this isn't needed.  Defaults to `False`.
+{% configuration %}
+domain:
+  description: This is the domain name to be used when sending out mail. Defaults to the first custom domain you have set up.
```

Comment:
Move "Defaults to the..." to the `default:` key

Reply:
You might be right, I'm not sure how to proceed with this.

Reply:
@frenck @fabaff What are we doing with this?

Reply:
Good question. It looks like that the configuration is made within mailgun thus it would help to replace `Defaults to` with something else. Maybe "Needs to be the first..."